dev-python/python-hkdf/python-hkdf-0.3.0.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8,9,10} )
+inherit distutils-r1
+
+DESCRIPTION="This module implements the HMAC Key Derivation function"
+HOMEPAGE="https://github.com/casebeer/python-hkdf"
+SRC_URI="https://github.com/casebeer/${PN}/archive/ba0e2eee8f50cc84706f816dbc57897319e2250c.zip -> ${P}.zip"
+
+S="${WORKDIR}/${PN}-ba0e2eee8f50cc84706f816dbc57897319e2250c"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND=""
+DEPEND="${RDEPEND}"
```

Comment:
- there's a tests.py we can run maybe, and I'd drop blank RDEPEND + DEPEND.
- I'd also consider setting a MY_COMMIT variable for use in SRC_URI + S.
- add BDEPEND="app-arch/unzip" for SRC_URI